src/com/google/cose/Ec2SigningKey.java

```diff
@@ -114,6 +118,74 @@ public ECPublicKey getPublicKey() {
     return (ECPublicKey) this.keyPair.getPublic();
   }
 
+  private static byte[] arrayFromBigNum(BigInteger n, int curveSize) {
```

Comment:
Nit: Please use a more descriptive name than `n`. I think `num` would suffice.

Reply:
renamed to `num`, done.